legend-engine-extensions-collection-generation/pom.xml

```diff
@@ -196,6 +196,13 @@
             <artifactId>legend-engine-xt-persistence-cloud-protocol</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-persistence-test-runner</artifactId>
+            <version>${project.version}</version>
```

Comment:
can we register the library under dependecyManagment, and avoid version here?

Reply:
> Approved - as discussed this feedback will be address on a subsequent MR.

@hardikmaheshwari @rafaelbey @epsstan Putting together PR to address all comments now.

Reply:
Done.